scripts/build_docker_image.sh

```diff
@@ -33,16 +33,15 @@ echo "source activate sqlflow-dev" >> ~/.bashrc
 source /miniconda/bin/activate sqlflow-dev && python -m pip install \
 numpy==1.16.1 \
 tensorflow==${TENSORFLOW_VERSION} \
-mysqlclient \
-impyla \
-pyodps \
-jupyter \
+mysqlclient==1.4.4 \
+impyla==0.16.0 \
+pyodps==0.8.3 \
+jupyter==1.0.0 \
 notebook==6.0.0 \
 sqlflow==0.5.0 \
-pre-commit \
-odps \
```

Comment:
I deleted odps package because it is duplicated to pyodps. Both of them are Python SDK for ODPS. I choose pyodps because [its PyPI page](https://pypi.org/project/pyodps) appears to be more official than [odps's PyPI page](https://pypi.org/project/odps/).

The odps package is originally introduced in #553. However, I don't see the necessity of it. @Yancey1989 @typhoonzero as the contribute of that PR, could you please confirm?